c7n/resources/glue.py

```diff
@@ -413,6 +414,47 @@ class resource_type(TypeInfo):
         cfn_type = 'AWS::Glue::SecurityConfiguration'
 
 
+@GlueSecurityConfiguration.filter_registry.register('kms-key')
+class KmsFilter(KmsRelatedFilter):
+    """
+    Filter a resource by its associcated kms key and optionally the aliasname
+    of the kms key by using 'c7n:AliasName'
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: glue-security-configuration-kms-key
+            resource: glue-security-configuration
+            filters:
+              - type: kms-key
+                key: c7n:AliasName
+                value: "^(alias/aws/)"
+                op: regex
+    """
+    schema = type_schema(
+        'kms-key',
+        key_type={'type': 'string', 'enum': [
```

Comment:
can we move this to the dictionary (`**`) as `key-type`.. everything else on this filter is hyphen-case, feels odd to make one field under_case.

Reply:
Yep, that makes way more sense.